app/controllers/admin/roles/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Admin::Roles
+  class UsersController < ApplicationController
+    include ActionView::Helpers::TextHelper
+    before_action :require_can_edit_roles!
+    before_action :set_role
+
+    def create
+      # add to the role any users passed through
+      user_ids = clean_params[:user_ids].select(&:present?).map(&:to_i)
+      @role.users = (@role.users + User.where(id: user_ids).to_a).uniq
```

Comment:
Let's move the logic here into `Role#add(users)`
within the add method, you can call `Array.wrap(users)` on the first line, then you could pass in a user or an array of users.

Reply:
Done.